packages/api/src/providers/oauth1.ts

```diff
@@ -174,8 +174,8 @@ export class OAuth1DefaultImpl implements OAuth1 {
         // DynamoDB (Webhook and auth)
         try {
           if (!cxId || cxId === userToken.cxId) {
-            capture.setUser({ id: userToken.cxId });
-            capture.setExtra({ id: userToken.userId });
+            capture.setUser({ id: userToken.userId });
```

Comment:
Sentry's `user` represents our cx, not our cx's user. So what we had before was the correct usage of it.

`userToken.userId` should be stored on `extra`, maybe `metriportUserId` for clarity?

Reply:
Ohhh, I didn't realize that Sentry's `user` was a customer! Now I see what you meant.

Reply:
I finally understand what you meant! I didn't realize that Sentry's `user` represented a cx!